.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+jobs:
+  python-3.4:
+    working_directory: ~/coala
```

Comment:
I think this isnt beneficial, because it needs to be repeated for each `job`, and would need to be different for each repo.

better to stick with the default. https://discuss.circleci.com/t/working-directory-is-no-longer-required-defaults-to-project/14363